app/models/incoming_message_error.rb

```diff
@@ -10,7 +10,6 @@
 #  retry_at   :datetime
 #  backtrace  :text
 #
-# models/incoming_message_error.rb:
```

Comment:
This looks like it's been removed in error - it wasn't originally added by annotate: https://github.com/mysociety/alaveteli/commit/eb944c6c7aebdd066ba897745155fb233335f789

Reply:
Ugh, why would annotate remove this. Think I'll add a blank line to prevent this from happening again.